app/views/events/_event.html.erb

```diff
@@ -1,11 +1,11 @@
 <div class="card">
-  <h1><%= link_to event.title.titleize, event %></h1>
+  <h1><%= link_to emojify(event.title.titleize), event %></h1>
   <div class="event-meta">
     <p><%= icon 'calendar-o', class: 'fa-fw' %><%= event.starts_at.strftime('%B %e, %Y  -  %k:%M') %></p>
     <p><%= icon 'map-marker', class: 'fa-fw' %><%= event.location.name %>, <a href="https://www.google.ca/maps/search/<%= event.location.address %>", target='_blank'><%= event.location.address %></a></p>
   </div>
   <div class="markdown-body">
-    <%= render_markdown_as_html event.body %>
+    <%= render_markdown_as_html emojify(event.body) %>
```

Comment:
I actually think makes sense to include the `emojify` helper within the `render_markdown_as_html` helper for two reasons:  

1. The way I see it github emoji i.e colon emoji colon syntax is part of the github markdown so it should be treated as such.
2.  Calling the `emojify` method shouldn't result in markdown being outputted, only the replacement of the :emoji: syntax with the right emoji, that doesn't seem like expected behavior.

 I see the `emojify`method as a helper method to the `render_markdown_as_html` method which replaces "markdown emoji syntax" with the right emoji.